hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/common/volume/VolumeInfoMetrics.java

```diff
@@ -104,23 +104,26 @@ public String getMetricsSourceName() {
    */
   @Metric("Returns the Used space")
   public long getUsed() {
-    return volume.getVolumeInfo().getScmUsed();
+    return volume.getVolumeInfo() != null ?
+            volume.getVolumeInfo().getScmUsed() : 0;
```

Comment:
nit: @ashishkumar50 Can we make volumeInfo field -> Optional<VolumeInfo> volumeInfo; it would be better for future development?

Reply:
I see there are other functions as well using this field. It would be better to change it to Optional.

Reply:
Changes to be made to the field volumeInfo in org.apache.hadoop.ozone.container.common.volume.StorageVolume;

Reply:
Hi @swamirishi , Thanks for review. I have updated volumeInfo to Optional.